mcp/gcp_ae.tf

```diff
@@ -468,4 +467,36 @@ resource "google_app_engine_standard_app_version" "self" {
   lifecycle {
     create_before_destroy = true
   }
+}
+
+resource "google_app_engine_service_split_traffic" "std-split" {
+  for_each = {
+    for service, specs in local.as_std_specs: service =>
```

Comment:
Can this not be in locals?

Reply:
Moved logic to locals, and included `depends_on` in the resources to ensure that traffic is assigned after creation/destruction of app engine version